src/main/java/io/confluent/examples/streams/utils/MonitoringInterceptorUtils.java

```diff
@@ -0,0 +1,41 @@
+package io.confluent.examples.streams.utils;
+
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.streams.StreamsConfig;
+
+import java.util.Properties;
+
+/**
+ * Utility helper class that will enable MonitoringInterceptors when a Kafka Streams application is deployed in Confluent Enterprise.
```

Comment:
Provide a link to our docs website that provides more information on Monitoring Interceptors

Reply:
ack